controller/src/main/java/org/jboss/as/controller/extension/SubsystemInformation.java

```diff
@@ -56,4 +58,25 @@
      * @return the micro interface version, or {@code null} if the subsystem does not have a versioned interface
      */
     Integer getManagementInterfaceMicroVersion();
+
+
+    /**
+     * Gets full version of the subsystem
+     * @return full {@link ModelVersion}
+     */
+    default ModelVersion getManagementInterfaceVersion(){
+        return ModelVersion.create(getManagementInterfaceMajorVersion() != null ? getManagementInterfaceMajorVersion() : 0,
+                getManagementInterfaceMinorVersion() != null ? getManagementInterfaceMinorVersion() : 0,
+                getManagementInterfaceMicroVersion() != null ? getManagementInterfaceMicroVersion() : 0
+        );
+    }
+
+
+    default boolean isDeprecated(){
+        return false;
+    }
```

Comment:
The getter is never used. The field in the implementation is also never read outside the getter. So the setter is pointless.

I don't want a pointless method added to an interface.